save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/validation/ValidationUtils.kt

```diff
@@ -13,7 +13,7 @@ const val NAMING_MAX_LENGTH = 22
 private val namingAllowedSpecialSymbols = setOf('-', '_', '.')
 
 @Suppress("MagicNumber")
-private val tagLengthRange = 3..15
+private val tagLengthRange = 2..15
```

Comment:
I think there was some purpose for “3” previously

Reply:
Ok. decided to change 3 to 2 only because of the `Go` tag